### PR TITLE
Add recipe 'Diana's lips'

### PR DESCRIPTION
**Pull request by lucasmartinelle**

### CODE REVIEW

File: src/recipes/dianas-lips.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        "Add the lime juice.",
+        "Complete with Schweppes Tonic Hibiscus.",
+        "Decorate your glass with slices of lime."
+    ],
+    "image": "dianas-lips.jpg",
+    "keywords": [
+        "Schweppes Tonic Hibiscus",
+        "Lemon",
+        "Alcohol",
+        "Mezcal"
```
Comment (alfg) [inline src/recipes/dianas-lips.json:39]: Can we lowercase these?

File: src/recipes/dianas-lips.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
+{
+    "name": "Diana's lips",
+    "description": "a cocktail based on mezcal infused with cocoa beans and pink berries, lime and Schweppes Tonic Hibiscus",
```
Comment (alfg) [inline src/recipes/dianas-lips.json:3]:
```suggestion
    "description": "A cocktail based on mezcal infused with cocoa beans and pink berries, lime and Schweppes Tonic Hibiscus.",

```

**Review by alfg — CHANGES_REQUESTED** [review]
Couple suggestions. Thanks.

**Comment by lucasmartinelle** [comment]
done

**Comment by alfg** [comment]
Thanks!

**Pull request merged**